app/src/main/java/org/sil/storyproducer/model/Workspace.kt

```diff
@@ -86,9 +86,25 @@ object Workspace{
 
     fun setupWorkspacePath(context: Context, uri: Uri) {
         try {
+            val outputDir: File = context.cacheDir
+            val outputFile = File.createTempFile("prefix", ".tmp", outputDir)
+            var newDoc : DocumentFile = DocumentFile.fromFile(outputFile)
+
+            // Issue 539 - Create a temporary file to become the new workspace
+            // Reset Story info to detach from old Story
+            workdocfile = newDoc
+
+            Stories.clear()
```

Comment:
Does Stories.clear() just empty the mutable list? Or does it process the current workdocfile for stories?

Reply:
It would just clear the list of Stories. After looking at it again, I don't think it is actually needed! It must happen again down the line. I will go ahead and remove it